content/Building a SPA web application using Hotwire - Matteo Guerrieri.md

```diff
@@ -0,0 +1,29 @@
+## Building a SPA web application using Hotwire
```

Comment:
Maybe it should be `Building a SPA using Hotwire`. Application twice seems redundant